client/layout/masterbar/minimal.jsx

```diff
@@ -3,24 +3,35 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React from 'react/addons';
 import Masterbar from './masterbar';
 
 /**
  * Internal dependencies
  */
 import Item from './item';
+import config from 'config';
 
-const MasterbarMinimal = ( { url } ) => (
-	<Masterbar>
-		<Item url={ url } icon="my-sites" className="masterbar__item-logo">
-			WordPress<span className="tld">.com</span>
-		</Item>
-	</Masterbar>
-);
+export default React.createClass( {
```

Comment:
> What is the rationale behind switching to React.createClass instead of keeping the stateless function syntax?

Only so that we can use the `this.translate()` call